### ISSUE TITLE
[BUG] Says EdXposed franework is active but none of the modules are working

### ISSUE THREAD

**Issue by Cyberavater** [ElderDrivers/EdXposed#780]
**什么情况/What happened?**

All modules are showed but none of them are working.

**Xposed模块列表/Xposed Module List**

Edxposed manager
FDex2
LP
NFG

**Magisk模块列表/Magisk Module List**

ADB and Fastboot
Busybox
Riru
Sony music player
UKA

**版本信息/Versions**

Android:10

Magisk: 21201

Riru: 23.1 (51)

EdXposed: 4655

**相关日志/Related Logs**

[Debug_Log.zip](https://github.com/ElderDrivers/EdXposed/files/5757305/Debug_Log.zip)

Versions older than 4598 works.

**Comment by kotori2** [comment]
All 3 modules loaded
```
12-31 18:43:36.203  2638  2638 I EdXposed-Bridge: Loading modules from /data/app/formatfa.xposed.Fdex2-p-v3krVCiYwzeiI99j6T0w==/base.apk
12-31 18:43:36.207  2638  2638 I EdXposed-Bridge:   Loading class formatfa.xposed.Fdex2.MainHook
12-31 18:43:36.212  2638  2638 I EdXposed-Bridge: Loading modules from /data/app/ru.MJEJLnXU.lFQXwrbfs-5gLXiOF-zkBLLL6G6Vq3-g==/base.apk
12-31 18:43:36.219  2638  2638 I EdXposed-Bridge:   Loading class com.xposed.XSupport
12-31 18:43:36.267  2638  2638 I EdXposed-Bridge: Loading modules from /data/app/nfg.multi_crack.android-wPCmOxw2VDr2787Wc_0L2g==/base.apk
12-31 18:43:36.274  2638  2638 I EdXposed-Bridge:   Loading class nfg.multi_crack.android.Xposed.XposedEntry
12-31 18:43:36.286  2638  2638 I EdXposed-Bridge: Loading modules from /data/app/org.meowcat.edxposed.manager-DBqu06Z3zGxTcxVQm-C7gQ==/base.apk
12-31 18:43:36.293  2638  2638 I EdXposed-Bridge:   Loading class org.meowcat.edxposed.manager.xposed.Enhancement
12-31 18:43:36.761  2638  2638 I EdXposed-Bridge: java.lang.ClassNotFoundException: com.android.dex.Dex
12-31 18:43:36.761  2638  2638 I EdXposed-Bridge: 没有指定apk,请打开模块选择要脱dex的apk
12-31 18:43:37.615  2151  2151 E EdXposed-Bridge: android.content.pm.PackageManager$NameNotFoundException: com.google.android.deskclock
12-31 18:43:37.615  2151  2151 E EdXposed-Bridge: 	at android.app.ApplicationPackageManager.getApplicationInfoAsUser(ApplicationPackageManager.java:414)
12-31 18:43:37.615  2151  2151 E EdXposed-Bridge: 	at android.app.ApplicationPackageManager.getApplicationInfo(ApplicationPackageManager.java:395)
12-31 18:43:37.615  2151  2151 E EdXposed-Bridge: 	at java.lang.reflect.Method.invoke(Native Method)
12-31 18:43:37.615  2151  2151 E EdXposed-Bridge: 	at com.swift.sandhook.SandHook.callOriginMethod(SandHook.java:183)
12-31 18:43:37.615  2151  2151 E EdXposed-Bridge: 	at com.swift.sandhook.xposedcompat.hookstub.HookStubManager.hookBridge(HookStubManager.java:380)
12-31 18:43:37.615  2151  2151 E EdXposed-Bridge: 	at SandHookerNew_3f68511eff97785e35a045b87022b7d6396f7fa8.hook(Unknown Source:58)
12-31 18:43:37.615  2151  2151 E EdXposed-Bridge: 	at com.android.launcher3.icons.clock.DynamicClock.getClockLayers(DynamicClock.java:81)
12-31 18:43:37.615  2151  2151 E EdXposed-Bridge: 	at com.android.launcher3.icons.clock.DynamicClock.lambda$updateMainThread$0$DynamicClock(DynamicClock.java:140)
12-31 18:43:37.615  2151  2151 E EdXposed-Bridge: 	at com.android.launcher3.icons.clock.-$$Lambda$DynamicClock$WVbs09_qyeFVbQF49BJHII9ccwI.run(Unknown Source:2)
12-31 18:43:37.615  2151  2151 E EdXposed-Bridge: 	at android.os.Handler.handleCallback(Handler.java:883)
12-31 18:43:37.615  2151  2151 E EdXposed-Bridge: 	at android.os.Handler.dispatchMessage(Handler.java:100)
12-31 18:43:37.615  2151  2151 E EdXposed-Bridge: 	at android.os.Looper.loop(Looper.java:214)
12-31 18:43:37.615  2151  2151 E EdXposed-Bridge: 	at android.app.ActivityThread.main(ActivityThread.java:7357)
12-31 18:43:37.615  2151  2151 E EdXposed-Bridge: 	at java.lang.reflect.Method.invoke(Native Method)
12-31 18:43:37.615  2151  2151 E EdXposed-Bridge: 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:493)
12-31 18:43:37.615  2151  2151 E EdXposed-Bridge: 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:925)
```

**Comment by Cyberavater** [comment]
All modules load but none of them actually works.
It works with the version older than 4598, can you suggest a specific module that I should use to test its functionality? @kotori2

**Comment by yujincheng08** [comment]
https://repo.xposed.info/module/com.jakting.fff

**Comment by Cyberavater** [comment]
> https://repo.xposed.info/module/com.jakting.fff

This module works. This module needs the device to restart **every time** to apply its effect, I think such modules will work fine. But can you suggest another module that won't need a reboot **every time** while using it? I.e a non-systemwide module maybe?

**Comment by yujincheng08** [comment]
https://repo.xposed.info/module/xyz.yingshaoxo.nomorechinese

**Comment by kotori2** [comment]
> This module works. This module needs the device to restart every time to apply its effect, I think such modules will work fine. But can you suggest another module that won't need a reboot every time while using it? I.e a non-systemwide module maybe?

EdXposed Manager itself is a Xposed Module, but it only hooks itself to show the green icon.

**Comment by Cyberavater** [comment]
Hmm, all other modules except these 3 (FDex2 LP NFG) seem to be working just fine.

You can try if this is working for you;
https://github.com/rockz5555/NFG-Multi-Crack/releases
(Only if you want to)

Nevertheless, thank you guys for your support.

**Comment by Cyberavater** [comment]
>You can try if this is working for you;
https://github.com/rockz5555/NFG-Multi-Crack/releases
(Only if you want to)

This will definitely work for you since you're installing it for the first time no need to test.

I just deduced the issue here;
Having the modules configured with previous EdXposed version made the issue. Uninstalling those 3 module apks and reinstalling them fixed the issue. This is a weird issue, but I just added this workaround here if this helps some other users facing similar issues.
